masonry/src/widget/sized_box.rs

```diff
@@ -26,7 +26,7 @@ use crate::{
 pub enum BackgroundBrush {
```

Comment:
My inclination would be to remove this type entirely, and just use a `Brush` from Peniko.

Reply:
Yeah, agree, I just see that `PaintCtx` is "leaked" in xilem with the `PainterFn`, which I don't think should be a thing?

Reply:
As it stands, I've been using the `PaintFn` variant specifically to show progress behind a nested view, like a watermark.

Reply:
So, if we remove the `PaintFn` variant what would be the idiomatic way to show a view over another?

Reply:
Isn't something like #553 more appropriate for such cases (when we have appropriate lay(er)outing possibilities)? (Or the direction I would rather like to go: having views similar as in the xilem_web svgtoy example)

Reply:
Yes, #553 would enable creating an ad-hoc watermark view. I was wondering about laying it out behind another view.